### ISSUE TITLE
Reach out to the Financial Sector

### ISSUE THREAD

**Issue by jaimiebutton** [bcgov/cas-public-reporting#100]
Woohoo! This is ticket #100.

**Acceptance criteria:**
- [x] Add contacts to Contact Tracker document
- [x] Reach out to project team for contact emails/introductions
- [x] F/U with Chris F
- [ ] Get confirmation from Chris

_The following Financial contacts have been identified:_
- BMO Chief Sustainability Officer
- RBC Director, Sustainable Finance
- Investors for Paris Compliance
- BCI Senior Managing Director ESG

**Comment by jaimiebutton** [comment]
Blocked by CF

**Comment by LindsayMacfarlane** [comment]
Note: moving to icebox. Unable to make contact with financial sector prior to research wrap up. @jaimiebutton - might be good to highlight as a future recommendation in the report.

**Comment by LindsayMacfarlane** [comment]
Note: not completed during initial research and discovery phase. Removed from External Research EPIC to capture in potential future work.

**Comment by jaimiebutton** [comment]
Closing this ticket as it's no longer relevant
